blackjax/mcmc_adaptation.py

```diff
@@ -0,0 +1,108 @@
+from typing import Callable, Union
+
+import jax
+
+from blackjax import hmc, nuts
+from blackjax.adaptation.window_adaptation import (
+    window_adaptation_base,
+    window_adaptation_schedule,
+)
+from blackjax.base import AdaptationAlgorithm
+from blackjax.types import Array, PRNGKey, PyTree
+
+__all__ = ["window_adaptation"]
+
+
+def window_adaptation(
```

Comment:
Nice. Is this function tested anywhere?

Reply:
That's the one used in `test_sampling.py`